text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,289 @@
+* Hopefully we can find a linter out there for this already
+
+### Search
+
+We can accomplish search using `Next.js` built in API system. Here is an [example](https://medium.com/@matswainson/building-a-search-component-for-your-next-js-markdown-blog-9e75e0e7d210https://medium.com/@matswainson/building-a-search-component-for-your-next-js-markdown-blog-9e75e0e7d210)
+search system using `Next.js`.
+
+### Change Log
+
+It would be nice to keep track of a `CHANGE_LOG` to see the week by week updates that
+happen to the respository. The functionality for this is mostly internal, and can serve
+as a method to expose newer board members to industry practices! We would most likely 
+have a `GitHub` action that runs at the time of dev team meeting each week, that would
+automatically create a pull request to update the `CHANGE_LOG`.
+
+## What are the requirements for this project?
+
+`xplr` will require developing in `Next.js`, writing GitHub Actions, building linters, 
+and deploying to Netlify. Ideally, `xplr` would be built across the CL dev team, but 
+realistically a single person should be able to build out most of the functionality.
+
+TLDR;
+* Learning the `Next.js` framework
+* Learning how to build a linter through `JavaScript`
+* Learning how to utilize GitHub Actions
+
+## What are we launching today?
+
+The entirety of this project seems pretty simple and shouldn't take too long to 
+develop. Future iterations can add custom metadata through the `gray-matter` of
+posts to add images to the metadata and improve SEO. However, everything I laid
+out in the blurb is reasonally achievable.
+
+## FAQ
+
+### How long should posts be?
+
+Inspo posts **should not** take excessive time to write and find. Exploration should
+be fun and hopefully this weekly task doesn't feel like a chore. Finding inspo and
+writing about it should take around 30 minutes. Writing concise documentation is a big
+part of developing and hopefully through this activity, our team will come to 
+appreciate documentation.
+
+### When would we start this program?
+
+Ideally, some time before Fall quarter to work out any kinks. Let's tentatively say 
+middle of September.
+
+### Would this program be utilized across all of Creative Labs?
+
+I think it would be nice for everyone on CL to look for inspiration on a daily basis.
+As the future tech director, I only have jurisdiction with my own team. But I would be
+willing to open this up to other teams to use as well.
+
+For example:
+* Design can list all the art/styles/sites that they come across
+* Marketing can list all the concepts/articles/excerpts they read
+
+**Note:** If we are to extend `xplr` to encompass all of CL in the future, it should
+have another RFC to explain the expansion and the reorganization of the respository.
+
+### What would the general workflow for a dev team member look like?
+
+1. Before the dev team meeting, each member will find something that inspires them and
+write a post about it. 
```

Comment:
Can these posts also be like a dev diary typa thing w some like code snippets and all that or should it be more high level

Reply:
I’ll add this to the FAQ, but I think some posts might not qualify for code snippets?

I’m thinking that whether or not the post has code snippets are not is up to the reviewer! If the reviewer is reading the post, and thinks that the post could be better with some code, then they should request that addition!

But I’m pretty flexible about the format and content of these posts! @sriramb2000 if you have another idea for a template we should totally add it to the RFC!